GVFS/GVFS.UnitTests/Common/Database/TableTests.cs

```diff
@@ -33,7 +33,7 @@ public void CreateTableTest()
             Mock<IDbConnection> mockConnection = new Mock<IDbConnection>(MockBehavior.Strict);
             mockConnection.Setup(x => x.CreateCommand()).Returns(mockCommand.Object);
 
-            this.CreateTable(mockConnection.Object);
+            this.CreateTable(mockConnection.Object, false);

```

Comment:
Style nit here and on line 53: `false` -> `caseSensitiveFileSystem: false`

Reply:
Fixed, thanks!